.github/workflows/sdk.yml

```diff
@@ -268,6 +268,61 @@ jobs:
         env:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+  
+  python-sdk-mac:
+    runs-on: macos-latest
+    env:
+      SQL_PYSDK_ENABLE: ON
+      OPENMLDB_BUILD_TARGET: 'cp_python_sdk_so openmldb'
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache thirdparty
+        uses: actions/cache@v2
+        with:
+          path: |
+            .deps/
+            thirdsrc
+          key: ${{ runner.os }}-thirdparty-${{ hashFiles('third-party/**/CMakeLists.txt', 'third-party/**/*.cmake', 'third-party/**/*.sh') }}
+
+      - name: prepare release
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        run: |
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+          VERSION=${VERSION#v}
+          bash steps/prepare_release.sh "$VERSION"
+
+      - name: build pysdk and sqlalchemy
+        run: |
+          make build
+
+      - name: prepare python deps
+        run: |
+          curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+          python3 get-pip.py
+          pip install setuptools wheel twine
+
+      - name: test sqlalchemy
+        run: |
+          bash steps/test_python.sh
+
+      - name: upload python ut results
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: mac-ut-result-python-${{ github.sha }}
```

Comment:
checkout `publish-test-results` job below, you might update `files` options there

Reply:
Change for two options of public result job.

Reply:
updated.